Connection/WSConnector.cs

```diff
@@ -23,11 +23,12 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Security.Authentication;
+using IBM.Cloud.SDK.Authentication;
 using System.Threading;
+using System;
 #if !NETFX_CORE
-using UnitySDK.WebSocketSharp;
+using IBM.Cloud.SDK.Plugins.WebSocketSharp;
```

Comment:
strange, if the namespace was always UnitySDK.WebSocketSharp we should not have been getting any conflicts. unless the other plugin renamed their namespace to UnitySDK as well! Probably good that we change this namespace though.